gtfs-realtime/CHANGES.md

```diff
@@ -29,7 +29,7 @@ When a producer or consumer is interested in adding a new field to the GTFS Real
     - Before calling for a vote, at least one GTFS-realtime producer and one GTFS-realtime consumer should implement the proposed change. It is expected that the GTFS-realtime producer(s) include the change in a public-facing GTFS-realtime feed and provide a link to that data within the pull request comments, and that the GTFS-realtime consumer(s) provides a link in the pull request comments to an application that is utilizing the change in a non-trivial manner (i.e, it is supporting new or improved functionality).
     - When calling for a vote, the advocate should clearly state whether the vote is for official adoption of the field into the spec or for an experimental field.
 1. Vote lasts the minimum period sufficient to cover 7 full calendar days and 5 full Swiss business days. Vote ends at 23:59:59 UTC.
-  	- The advocate should announce the specific end time at the start of the vote.
+  	- The advocate should announce the specific end time at the start of the vote - both on the pull request and on the [GTFS Changes mailing list](https://groups.google.com/forum/#!forum/gtfs-changes).
```

Comment:
For GTFS-rt, it should be `[GTFS Realtime mailing list](https://groups.google.com/forum/#!forum/gtfs-realtime)`

Reply:
Done!